substrate/frame/offences/src/tests.rs

```diff
@@ -22,10 +22,31 @@
 use super::*;
 use crate::mock::{
 	new_test_ext, offence_reports, with_on_offence_fractions, Offence, Offences, RuntimeEvent,
-	System, KIND,
+	System, Runtime, KIND,
 };
 use frame_system::{EventRecord, Phase};
 use sp_runtime::Perbill;
+use sp_core::H256;
+
+#[test]
+fn should_get_reports_with_storagemap_getter() {
+	new_test_ext().execute_with(|| {
+		// given
+		let report_id: ReportIdOf<Runtime> = H256::from_low_u64_be(1);		
+		let offence_details = OffenceDetails {
+			offender: 1,
+			reporters: vec![2, 3],
+		};
+
+		Reports::<Runtime>::insert(report_id, offence_details.clone());
+		
+		// when
+		let stored_offence_details = Reports::<Runtime>::get(report_id);
+
+		// then
```

Comment:
```suggestion
		#[allow(deprecated)]
		let stored_offence_details = Offences::reports(report_id);
		assert_eq!(stored_offence_details, Some(offence_details.clone()));
		
		let stored_offence_details = Reports::<Runtime>::get(report_id);
```
This new part of the test should pass to avoid a breaking change

Reply:
Thanks. Done. Only slightly differently.